components/tests/about/divisions.test.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DivisionCard from '../../src/common/cards/division-card';
+import { Language, ContentModel } from '../../src/types';
+
+
+type CheckButtonProps = {
+    text: Record<Language, string>;
+    href: string;
+};
+
+export const mockStaticImageData = {
+    src: "/some-image.png",
+    height: 100,
+    width: 100,
+    blurDataURL: '',
+} as unknown as import('next/image').StaticImageData;
```

Comment:
Lets add import fot StaticImageDate and replace this with:
```suggestion
} as unknown as StaticImageData;
```

Reply:
Just pushed changes.